src/main/interfaces/RoleItem.ts

```diff
@@ -0,0 +1,5 @@
+export interface RoleItem {
+  name: string;
+  visible: boolean;
+  row: JQuery;
+}
```

Comment:
This file can be removed now as we're not using the type anywhere (I don't think?)